content/en/user-guide/chaos-engineering/chaos-application-dashboard/index.md

```diff
@@ -18,7 +18,7 @@ The dashboard uses [LocalStack Chaos API]({{< ref "chaos-api" >}}) under the hoo
 You can find this feature in the LocalStack Web Application by navigating to [**app.localstack.cloud/chaos-engineering**](https://app.localstack.cloud/chaos-engineering).
 
 {{< callout "note" >}}
-Chaos Engineering Dashboard is offered as a **preview** feature and under active development.
+Chaos Engineering Dashboard is offered is offered as a **preview** feature and is under active development.
```

Comment:
oh wait... why do we repeat "is offered is offered" so many times in this PR? It looks like a mistake... what am I missing, @HarshCasper?  😆